### PR TITLE
Add `getAddressFromScriptPubKey` function

### PR DESCRIPTION
**Pull request by r-czajkowski**
~Depends on: #632~

Add the new function that returns the Bitcoin address based on the script pub key placed on the output of a Bitcoin transaction.

### CODE REVIEW

File: typescript/src/bitcoin.ts [hunk 1]
```diff
@@ -624,3 +624,20 @@ export function locktimeToNumber(locktimeLE: Buffer | string): number {
...
+
+/**
+ * Returns the Bitcoin address based on the script pub key placed on the output
+ * of a Bitcoin transaction.
+ * @param scriptPubKey Scirpt pub key placed on the output of a Bitcoin
+ *        transaction.
+ * @param network Bitcoin network.
+ * @returns The Bitcoin address.
+ */
+export function getAddressFromScriptPubKey(
```
Comment (lukasz-zimnoch) [inline typescript/src/bitcoin.ts:636]: The `scriptPubKey` is a field specific to transaction outputs but basically, this function converts any script to a Bitcoin address. We can make this function more generic and explicitly the opposite of `createOutputScriptFromAddress`. That said, I propose naming it `createAddressFromOutputScript`.

File: typescript/src/bitcoin.ts [hunk 2]
```diff
@@ -624,3 +624,20 @@ export function locktimeToNumber(locktimeLE: Buffer | string): number {
...
+  scriptPubKey: string,
```
Comment (lukasz-zimnoch) [inline typescript/src/bitcoin.ts:637]:
```suggestion
  script: Hex,
```

File: typescript/src/bitcoin.ts [hunk 3]
```diff
@@ -624,3 +624,20 @@ export function locktimeToNumber(locktimeLE: Buffer | string): number {
 export function createOutputScriptFromAddress(address: string): Hex {
   return Hex.from(Script.fromAddress(address).toRaw().toString("hex"))
 }
+
+/**
+ * Returns the Bitcoin address based on the script pub key placed on the output
+ * of a Bitcoin transaction.
+ * @param scriptPubKey Scirpt pub key placed on the output of a Bitcoin
+ *        transaction.
```
Comment (lukasz-zimnoch) [inline typescript/src/bitcoin.ts:632]:
```suggestion
 * @param script The unprefixed and not prepended with length output script.
```

File: typescript/src/bitcoin.ts (hunk 1 above)
Comment (r-czajkowski) [reply typescript/src/bitcoin.ts]: c6f94f6

File: typescript/src/bitcoin.ts (hunk 2 above)
Comment (r-czajkowski) [reply typescript/src/bitcoin.ts]: 1c9b91b

File: typescript/src/bitcoin.ts (hunk 3 above)
Comment (r-czajkowski) [reply typescript/src/bitcoin.ts]: 1c9b91b

**Review by lukasz-zimnoch — APPROVED** [review]

**Pull request merged**